packages/snaps-controllers/src/snaps/SnapController.test.tsx

```diff
@@ -615,7 +616,17 @@ describe('SnapController', () => {
       getSnapControllerOptions({
         messenger,
         state: {
-          snaps: getPersistedSnapsState(),
+          snaps: getPersistedSnapsState(
+            getPersistedSnapObject({
+              manifest: {
+                initialConnections: {
+                  // @ts-expect-error test mock
```

Comment:
What error is expected here?

Reply:
Error that editor is showing. A bit annoying to have it there, but removed it since it doesn't have any impact on the test run I think 🤔 

```
TS2322: Type
{   'https://snaps.metamask.io': {};   'https://portfolio.metamask.io': {}; }
is not assignable to type Record<string & URL, {}>

Object literal may only specify known properties, and 'https://snaps.metamask.io' does not exist in type Record<string & URL, {}>
```